### PR TITLE
Added Query EMR Security Configuration Encryptions Enabled

### LINKED ISSUE #1071: Add EMR Security Configuration Encryptions Enabled query AWS Cloud Formation
### Platform
AWS

### Provider
AWS Cloud Formation

### Description
EMR SecurityConfiguration should enable and properly configure encryption at rest and in transit.

### PR DESCRIPTION
**Pull request by cataraujo190**
Description
EMR SecurityConfiguration should enable and properly configure encryption at rest and in transit.

Close #1071

### CODE REVIEW

File: assets/queries/cloudFormation/emr_security_configuration_encryptions_enabled/metadata.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "emr_security_configuration_encryptions_enabled",
+  "queryName": "EMR Security Configuration Encryptions Enabled",
+  "severity": "MEDIUM",
+  "category": null,
+  "descriptionText": "EMR SecurityConfiguration should enable and properly configure encryption at rest and in transit.",
+  "descriptionUrl": "https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-emr-securityconfiguration.html"
+}
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/emr_security_configuration_encryptions_enabled/metadata.json:8]: [As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline

File: assets/queries/cloudFormation/emr_security_configuration_encryptions_enabled/test/negative.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+---
+Resources:
+  EMRSecurityConfiguration-02:
+    Type: AWS::EMR::SecurityConfiguration
+    Properties:
+       Name: String
+       SecurityConfiguration:
+         EncryptionConfiguration:
+           EnableInTransitEncryption: true
+           EnableAtRestEncryption: true
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/emr_security_configuration_encryptions_enabled/test/negative.yaml:39]: [As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline

File: assets/queries/cloudFormation/emr_security_configuration_encryptions_enabled/test/negative.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+           AtRestEncryptionConfiguration:
+             LocalDiskEncryptionConfiguration:
+                 EnableEbsEncryption: true
+                 EncryptionKeyProviderType: AwsKms
+                 AwsKmsKey: arn:aws:kms:us-east-1:123456789012:key/12345678-1234-1234-1234-123456789012
+---
+Resources:
+  EMRSecurityConfiguration-01:
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/emr_security_configuration_encryptions_enabled/test/negative.yaml:19]: Remove `-` : `E3006 Resource EMRSecurityConfiguration-01 has invalid name.  Name has to be alphanumeric.`

File: assets/queries/cloudFormation/emr_security_configuration_encryptions_enabled/test/negative.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+       SecurityConfiguration:
+         EncryptionConfiguration:
+           AtRestEncryptionConfiguration:
+             LocalDiskEncryptionConfiguration:
+                 EnableEbsEncryption: true
+                 EncryptionKeyProviderType: AwsKms
+                 AwsKmsKey: arn:aws:kms:us-east-1:123456789012:key/12345678-1234-1234-1234-123456789012
+---
+Resources:
+  EMRSecurityConfiguration-02:
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/emr_security_configuration_encryptions_enabled/test/negative.yaml:32]: Remove `-` :
`E3006 Resource EMRSecurityConfiguration-02 has invalid name.  Name has to be alphanumeric.`

File: assets/queries/cloudFormation/emr_security_configuration_encryptions_enabled/test/positive.yaml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+           EnableInTransitEncryption: false
+           EnableAtRestEncryption: false
+           AtRestEncryptionConfiguration:
+             LocalDiskEncryptionConfiguration:
+                 EnableEbsEncryption: true
+                 EncryptionKeyProviderType: AwsKms
+                 AwsKmsKey: arn:aws:kms:us-east-1:123456789012:key/12345678-1234-1234-1234-123456789012
+---
+Resources:
+  EMRSecurityConfiguration-01:
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/emr_security_configuration_encryptions_enabled/test/positive.yaml:18]: Remove `-` :
`E3006 Resource EMRSecurityConfiguration-01 has invalid name.  Name has to be alphanumeric.`

File: assets/queries/cloudFormation/emr_security_configuration_encryptions_enabled/test/positive.yaml [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+    Properties:
+       Name: String
+       SecurityConfiguration:
+         EncryptionConfiguration:
+           AtRestEncryptionConfiguration:
+             LocalDiskEncryptionConfiguration:
+                 EnableEbsEncryption: false
+---
+Resources:
+  EMRSecurityConfiguration-03:
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/emr_security_configuration_encryptions_enabled/test/positive.yaml:29]: Remove `-` :
`E3006 Resource EMRSecurityConfiguration-03 has invalid name.  Name has to be alphanumeric.`

File: assets/queries/cloudFormation/emr_security_configuration_encryptions_enabled/metadata.json [hunk 7]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "emr_security_configuration_encryptions_enabled",
+  "queryName": "EMR Security Configuration Encryptions Enabled",
+  "severity": "MEDIUM",
+  "category": null,
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/emr_security_configuration_encryptions_enabled/metadata.json:5]: `"category": "Encryption and Key Management"`

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]
Almost good

**Comment by rogeriopeixotocx** [comment]
LGTM ✅

**Review by ruigomescx — APPROVED** [review]

**Pull request merged**